lib/converters/v2.0.0/converter-v2-to-v1.js

```diff
@@ -363,10 +363,11 @@ _.assign(Builders.prototype, {
                 _.isArray(v2Auth[v2Auth.type]) && (v2Auth = util.authArrayToMap(req));
                 helperAttributes = util.authMappersFromCurrent[currentHelper](v2Auth[v2Auth.type]);
             }
-            else {
-                helperAttributes = {};
-            }
         }
+        else if (v2Auth === null) {
+            currentHelper = null;
```

Comment:
Shouldn't we explicitly set `helperAttributes` to `null` as well in this case?

Reply:
No, `helperAttributes` are not present if `currentHelper` is `null`.

Reply:
Okay I remember, this is to be handled as a special case in Sync.